web-app/src/main/java/com/google/sps/servlets/ReportServlet.java

```diff
@@ -32,14 +38,17 @@
 @WebServlet("/report")
 public class ReportServlet extends HttpServlet {
 
-  private Map<String, String> PARAM_DEFAULT_MAP = new HashMap<String, String>(){{
+  private Map<String, String> PARAM_DEFAULT_MAP = new HashMap<String, String>() {
+    {
       put("title", "");
       put("latitude", "0.0");
       put("longitude", "0.0");
       put("timestamp", "0");
       put("incidentType", "etc");
-      put("description", ""); 
-  }};
+      put("description", "");
+      put("image", "");
```

Comment:
I think you deal with null case seperately with getUploadedFileUrl function, so there is no need to add default value here